best-practices/README.md

```diff
@@ -264,6 +264,21 @@ Objective-C
 * Prefer categories on `Foundation` classes to helper methods.
 * Prefer string constants to literals when providing keys or key paths to methods.
 
+Android
+-------
+
+* Each callback on an `Activity` should invoke methods on an instance of a
```

Comment:
I realize this isn't how the rest of the `best-practices` is organized but I think we should separate `project and style` guidelines from`architecture` guidelines as those are very different and the `architecture` one is more likely to change. I also think that helps give readers an idea of what type of "best practice" were talking about. Because for example, "unit test each presenter object" only makes sense if the user is using the MVP architecture pattern. 

For formal guidelines I happen to really love [Ribot labs' guidelines](https://github.com/ribot/android-guidelines). I think they did an excellent job and at some points maybe even get too specific. But I kinda love that. 

Maybe we update those to our standards and add some Kotlin stuff instead?